<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0" rowminimumheight="0,10,0">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxServerClient">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="pageServer">
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,10,10,0,0" columnminimumwidth="0,0,10,10,0,0">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonServerTcpIp">
            <property name="text">
             <string>TCP/IP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButtonServerUdp">
            <property name="text">
             <string>UDP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEditServerIpNum"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditServerPortNum"/>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pushButtonServer">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButtonIpList">
            <property name="text">
             <string>IP List</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageClient">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0" columnstretch="0,0,10,10,0" columnminimumwidth="0,0,10,10,0">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonClientTcpIp">
            <property name="text">
             <string>TCP/IP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButtonClientUdp">
            <property name="text">
             <string>UDP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEditClientIpNum"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditClientPortNum"/>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pushButtonClient">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="listViewConversation"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="hL02" stretch="10,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConfirm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
